setup.py

```diff
@@ -225,14 +225,7 @@ def _get_arrow_lib_as_linker_input(self):
     package_dir={
         "": "src",
     },
-    package_data={
-        "snowflake.connector": [
-            "*.md",
-            "LICENSE.txt",
-            "NOTICE",
-            "src/snowflake/connector/py.typed",
-        ],
-    },
+    include_package_data=True,
```

Comment:
This gets the list of files from `MANIFEST.in` instead of having a second list here in `setup.py. Feel free to look at the [docs](https://setuptools.pypa.io/en/latest/userguide/datafiles.html#data-files-support) for more info